### PR TITLE
Add v3 WAC

### LINKED ISSUE #886: Add the WAC v3 global mosaic to the WMS servers that removes the black stripe on the backside

### PR DESCRIPTION
**Pull request by alexanderbock**
Closes #886

### CODE REVIEW

**Comment by alexanderbock** [comment]
- Should remove the old map
 - Rename WAC3 -> WAC

File: data/assets/scene/solarsystem/planets/earth/moon/layers/colorlayers/wac_sweden.asset [hunk 1]
```diff
@@ -4,14 +4,9 @@ local layer = {
...
-    and February 2011. The WAC maps the whole Moon in one month, however the solar
-    incidence angle at the equator changes about 28° from the beginning to the end of the
-    month. To even out the incidence angle variations (generally angles between 55-75
-    degrees), this morphology mosaic (at 643 nm), is comprised of data collected over
-    three periods (1/20/2010 to 1/28/2010, 5/30/2010 to 6/6/2010, 7/24/2010 to 7/31/2010).
+  Description = [[LLunar Reconnaissance Orbiter Camera (LROC) Wide Angle Camera (WAC) aboard the Lunar Reconnaissance Orbiter (LRO) has allowed the instrument team to create a global mosaic comprised of over 15,000 images acquired between November 2009 and February 2011. The WAC maps the whole Moon in one month, however the solar incidence angle at the equator changes about 28° from the beginning to the end of the month. To even out the incidence angle variations (generally angles between 55-75 degrees), this morphology mosaic (at 643 nm), is comprised of data collected over three periods (1/20/2010 to 1/28/2010, 5/30/2010 to 6/6/2010, 7/24/2010 to 7/31/2010).
```
Comment (WeirdRubberDuck) [inline data/assets/scene/solarsystem/planets/earth/moon/layers/colorlayers/wac_sweden.asset:7]: Extra L in Lunar, beginning of first sentence.

File: data/assets/scene/solarsystem/planets/earth/moon/layers/colorlayers/wac_utah.asset [hunk 2]
```diff
@@ -4,17 +4,12 @@ local layer = {
   Identifier = "WAC_Utah",
   Name = "WAC [Utah]",
   Enabled = asset.enabled,
+  Description = [[LLunar Reconnaissance Orbiter Camera (LROC) Wide Angle Camera (WAC) aboard the Lunar Reconnaissance Orbiter (LRO) has allowed the instrument team to create a global mosaic comprised of over 15,000 images acquired between November 2009 and February 2011. The WAC maps the whole Moon in one month, however the solar incidence angle at the equator changes about 28° from the beginning to the end of the month. To even out the incidence angle variations (generally angles between 55-75 degrees), this morphology mosaic (at 643 nm), is comprised of data collected over three periods (1/20/2010 to 1/28/2010, 5/30/2010 to 6/6/2010, 7/24/2010 to 7/31/2010).
```
Comment (WeirdRubberDuck) [inline data/assets/scene/solarsystem/planets/earth/moon/layers/colorlayers/wac_utah.asset:7]: Same here, double L

File: data/assets/scene/solarsystem/planets/earth/moon/layers/colorlayers/wac_utah.asset [hunk 3]
```diff
@@ -31,7 +26,7 @@ asset.export("layer", layer)
 
 asset.meta = {
   Name = "WAC [Utah]",
-  Version = "1.1",
+  Version = "1.0",
```
Comment (WeirdRubberDuck) [inline data/assets/scene/solarsystem/planets/earth/moon/layers/colorlayers/wac_utah.asset:29]: This is not correct? (reducing the version number). Should it be  2.0?

**Review by WeirdRubberDuck — APPROVED** [review]

File: data/assets/scene/solarsystem/planets/earth/moon/layers/colorlayers/wac_sweden.asset (hunk 1 above)
Comment (alexanderbock) [reply data/assets/scene/solarsystem/planets/earth/moon/layers/colorlayers/wac_sweden.asset:7]: Yea, the text is copied from https://astrogeology.usgs.gov/search/map/Moon/LRO/LROC_WAC/Lunar_LRO_LROC-WAC_Mosaic_global_100m_June2013.  It does say "(Description from URL)" but that line jumped one too far down

File: data/assets/scene/solarsystem/planets/earth/moon/layers/colorlayers/wac_utah.asset (hunk 3 above)
Comment (alexanderbock) [reply data/assets/scene/solarsystem/planets/earth/moon/layers/colorlayers/wac_utah.asset:29]: Indeed

**Pull request merged**